src/App.tsx

```diff
@@ -1,61 +1,40 @@
+import React, { useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import { NewTodoForm } from './components/newTodoForm/newTodoForm';
+import { Todo } from './types';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+const getPreperedTodos = (todos: Todo[]) => {
```

Comment:
There is a typo in the function name `getPreperedTodos`. It should be `getPreparedTodos` to maintain correct spelling and readability.